Tests/SentryTests/SentrySpanContextTests.swift

```diff
@@ -0,0 +1,43 @@
+import XCTest
+
+class SentrySpanContextTests: XCTestCase {
+   
+    func testInitWithSampled() {
+        let spanContext = SpanContext(sampled: true)
+        XCTAssertTrue(spanContext.sampled)
```

Comment:
One could argue that if the test is verifying `sampled`, it doesn't need to break if something else changes.

And other tests could be added for other fields/use cases.

I'd add: `testInitDefaultValues` and in that test verify what default values should look like.

Otherwise if you change the default value, say, SpanID from 16 char (or whatever) to 30, all tests will break.